karma.config.js

```diff
@@ -73,26 +73,18 @@ module.exports = function(karma) {
                     'importSource': '@bpmn-io/properties-panel/preact',
                     'runtime': 'automatic'
                   } ]
-                ]
+                ].concat(coverage ?
+                  [ [ 'istanbul', { exclude: [
+                    'test/**/*'
```

Comment:
We want to exclude tests from coverage report because we care only for the source code coverage. Try this out yourself with `COVERAGE=1 npm t && open dist/lcov-report/index.html`.

Reply:
Ah sorry, I completely missed it's inside `exclude`. I just updated the same for `form-js` and was kind of blind for this 😆